build-tools/xaprepare/xaprepare/ThirdPartyNotices/Xamarin.Android.Build.Tasks.cs

```diff
@@ -58,42 +57,6 @@ class XamarinAndroidBuildTasks_fsharp_fsharp_TPN : ThirdPartyNotice
 		public override Uri    SourceUrl   => url;
 	}
 
-	class XamarinAndroidBuildTasks_fsprojects_FSharpCompilerCodeDom_TPN : ThirdPartyNotice
-	{
-		static readonly Uri    url         = new Uri ("https://github.com/fsprojects/FSharp.Compiler.CodeDom/");
-
-		public override string LicenseFile => null;
-		public override string Name        => "fsprojects/FSharp.Compiler.CodeDom";
-		public override Uri    SourceUrl   => url;
-
-		public override string LicenseText => @"
-        This is free and unencumbered software released into the public domain.
-
-        Anyone is free to copy, modify, publish, use, compile, sell, or
-        distribute this software, either in source code form or as a compiled
-        binary, for any purpose, commercial or non-commercial, and by any
-        means.
-
-        In jurisdictions that recognize copyright laws, the author or authors
-        of this software dedicate any and all copyright interest in the
-        software to the public domain. We make this dedication for the benefit
-        of the public at large and to the detriment of our heirs and
-        successors. We intend this dedication to be an overt act of
-        relinquishment in perpetuity of all present and future rights to this
-        software under copyright law.
-
-        THE SOFTWARE IS PROVIDED ""AS IS"", WITHOUT WARRANTY OF ANY KIND,
-        EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
-        MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
-        IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
-        OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
-        ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
-        OTHER DEALINGS IN THE SOFTWARE.
-
-        For more information, please refer to <http://unlicense.org>
-";
-	}
-
```

Comment:
If you build this branch locally, there should be a `ThirdPartyNotices.txt` we commit, too. Sometimes we forget to update it.

Reply:
@jonathanpeppers is this in https://github.com/xamarin/xamarin-android/blob/master/ThirdPartyNotices.txt? It looks like this file currently doesn't have any mention of `FSharp.Compiler.CodeDom`...

My local build didn't show any diffs in this file, maybe I missed a step? Or maybe this content was never included? (cc @grendello).

Reply:
Maybe some of these notices aren't making it to the file on disk? 👀

Reply:
I added an issue to track this https://github.com/xamarin/xamarin-android/issues/4179. I am not sure if we need to address that audit as part of this PR.